src/actions/assetsActions.js

```diff
@@ -556,15 +543,21 @@ export const fetchAllAccountsTotalBalancesAction = () => {
 
 export const fetchAssetsBalancesAction = () => {
   return async (dispatch: Dispatch, getState: GetState) => {
-    const { accounts: { data: accounts } } = getState();
+    const {
+      accounts: { data: accounts },
+      assetsBalances: { isFetching },
+      session: { data: { isOnline } },
+    } = getState();
 
     const activeAccount = getActiveAccount(accounts);
-    if (!activeAccount) return;
+    if (!activeAccount || isFetching || !isOnline) return;
```

Comment:
Do we want to copy the same check to `fetchAllAccountsAssetsBalancesAction`

Reply:
Copied on multi chain rates PR https://github.com/pillarwallet/pillarwallet/pull/2729/files#diff-7e53eac260989ebf191a1c88a9455b9c514ccd2dde73a6f538f1c55da55f0274R592.